src/v/redpanda/admin/cluster_config_schema_util.cc

```diff
@@ -81,5 +83,5 @@ util::generate_json_schema(const config::configuration& conf) {
 
     std::map<ss::sstring, property_map> response = {
       {ss::sstring("properties"), std::move(properties)}};
-    return ss::json::json_return_type(std::move(response));
+    return ss::json::stream_object(std::move(response));
```

Comment:
Do we ever want json_return_type? Is this always a safe swap?

Reply:
I think in most of the cases probably not. I haven't fully checked whether there is totally parity between the two or whether there is still some more unimplemented things on the seastar side.

It should always be safe. `stream_object` makes a copy. So this is possibly one case where you might need the former if you have an uncopyable or unmovable object.

Reply:
> whether there is still some more unimplemented things on the seastar side.

The kind of unimpl things you are thinking about will result in compile-time errors, right?

Reply:
Yeah exactly, or worst case fall back to the `to_json` (convert to string) impl

Reply:
I was going to say this would be a good thing to communicate out, but seems like you already have that it mind.